docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md

```diff
@@ -0,0 +1,31 @@
+# Architecture overview
+
+A Nebula Graph cluster consists of three services: the Graph Service, the Storage Service, and the Meta Service.
+
+Each service has its executable binaries and processes launched from the binaries. You can deploy a Nebula Graph cluster on a single machine or multiple machines using these binaries.
+
+The following figure shows the logical architecture of a typical Nebula Graph cluster.
+
+![Nebula Graph architecture](https://docs-cdn.nebula-graph.com.cn/docs-2.0/1.introduction/2.nebula-graph-architecture/nebula-graph-architecture2.png "Nebula Graph architecture")
+
+## The Meta Service
+
+The Meta Service in the Nebula Graph architecture is supported by a cluster composed of nebula-metad processes. It is responsible for metadata storage and processing and tasks such as CRUD operations on schema structures, cluster administration, and user privilege management.
```

Comment:
. It is responsible for metadata storage, such as  schema operations, cluster administration, and user privilege.